### PR TITLE
Update dotnet.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
This took a long time to find when using `Down`.

Per MS style guide avoid parenthetical expressions (they're hard to read in English (and impossible (nearly) to MT (Machine Translate))) :)

### CODE REVIEW

File: entity-framework/core/cli/dotnet.md [hunk 1]
```diff
@@ -268,13 +268,13 @@ The [common options](#common-options) are listed above.
...
 
-Removes the last migration (rolls back the code changes that were done for the migration).
+Removes the last migration by calling the `Down` method of the latest migration. The command rolls back the code changes that were done for the latest migration.
 
 Options:
 
 | Option                 | Short             | Description                                                                     |
 |:-----------------------|:------------------|:--------------------------------------------------------------------------------|
-| <nobr>`--force`</nobr> | <nobr>`-f`</nobr> | Revert the migration (roll back the changes that were applied to the database). |
+| <nobr>`--force`</nobr> | <nobr>`-f`</nobr> | Revert the latest migration, that is, rolls back the changes that were applied to the database on the last migration. |
```
Comment (roji) [inline entity-framework/core/cli/dotnet.md:277]: How about:

```suggestion
| <nobr>`--force`</nobr> | <nobr>`-f`</nobr> | Revert the latest migration, rolling back the changes that were applied to the database in the last migration. |
```

**Review by roji — APPROVED** [review]

**Comment by ajcvickers** [comment]
@bricelam Is this change correct? Removing a migration doesn't cause `Down` to be called, does it?

**Comment by bricelam** [comment]
This part isn't correct:

> Removes...by calling the `Down` method

We only run the operations described in the `Down` method if the user tells us to. (By confirmation or by specifying `--force`) And, it's not related to rolling back the code changes.

**Comment by roji** [comment]
Sorry, I was focusing on the language here and missed this.

Submitted #3156 to fix this, @bricelam is that correct?

FWIW I know we've discussed this before, though I still find it odd for `--force` to do something different from what the command does without it (with it we touch the database, without it we don't). From my Unix mindset, `--force` is supposed to do the exact same thing, just to force the tool to complete silently without aborting on risky situations. Hopefully I'm not still misunderstanding.

**Comment by bricelam** [comment]
`--force` will skip checking the database. This might in turn skip reverting the migration. But we also warn during `--force` that you might have to manually revert the database if the migration was applied. In other words, `--force` avoids some built-in guardrails that might be preventing you from removing the C# code for the migrations.

**Pull request merged**